rust/src/filecontainer.rs

```diff
@@ -103,15 +65,28 @@ pub(super) unsafe fn FileOpenFileWithId(
     0
 }
 
-impl FileContainer {
-    pub fn free(&mut self, cfg: &'static SuricataFileContext) {
+pub trait FileContainerWrapper {
```

Comment:
suggested by rustc : since I cannot implement a type defined outside the crate, I define a trait and implement it for the type outside the crate...